packages/fhir-team-management/src/components/TeamsAddEdit/index.tsx

```diff
@@ -86,10 +87,15 @@ export const TeamsAddEdit: React.FC<Props> = (props: Props) => {
           practitioners: practitionerInfo.map((practitioner) => practitioner.id),
         });
       })
-      .catch(() => sendErrorNotification(lang.ERROR_OCCURRED));
+      .catch(() => {
+        setApiError(true);
+        sendErrorNotification(lang.ERROR_OCCURRED);
+      });
   }
 
-  if (!practitioners.data || (params.id && (!initialValue || !allRoles.data)))
+  if (team.isError || allRoles.isError || practitioners.isError || apiError) return <BrokenPage />;
+
+  if (!practitioners.isSuccess || (params.id && (!initialValue || !allRoles.isSuccess)))
```

Comment:
ref: https://github.com/opensrp/web/pull/887/files#r756834336
in this case should we show the loader, when say `query.isSuccess` is false but `data` is hydrated

Reply:
https://github.com/opensrp/web/pull/887#discussion_r761711167